app/(tabs)/index.tsx

```diff
@@ -1,11 +1,117 @@
-import { View, Text } from "react-native";
+import { View, Text, StyleSheet, ScrollView, FlatList } from "react-native";
+import ConfirmationCard from "../components/confirmationCard";
+import AppointmentCard from "../components/appointmentCard";
+import Colors from "../utils/Colors";
+
+const props = {
+    name: "Stephan"
+}
+
+const Data = [
+    {
+        id: "1",
+        subject: "Nachhilfe mit ",
+        name: "Leon",
+        headline: "Bestätigung",
+        body: "Es wurde um eine Bescheinigung für eure Lernunterstützung gebeten. Nur wenn du dies bestätigst, können wir diesen Wunsch erfüllen.",
+        linkText: "Angaben bestätigen"
+    },
+    {
+        id: "2",
+        subject: "Lernfortschritt mit ",
+        name: "Anna",
+        headline: "Erfolg bestätigen",
+        body: "Anna hat große Fortschritte gemacht. Bitte bestätige ihre Leistung, damit wir ihr eine entsprechende Bescheinigung ausstellen können.",
+        linkText: "Leistung bestätigen"
+    },
+    {
+        id: "3",
+        subject: "Feedback für ",
+        name: "Max",
+        headline: "Rückmeldung",
+        body: "Max hat um eine Rückmeldung zu seinen letzten Lernstunden gebeten. Bitte bestätige, dass du seine Leistung bewerten möchtest.",
+        linkText: "Feedback geben"
+    }
+    
+];
 
 const StartPage = () => {
     return (
-        <View>
-            <Text>Home Page</Text>
-        </View>
+        <ScrollView style={styles.pageContainer} showsVerticalScrollIndicator={false}>
+            <View style={styles.greeting}>
+                <Text style={styles.greetingText}>Hallo {props.name}!</Text>
+            </View>
+            <View style={{height: 40}}/>
+            <View style={styles.nextSteps}>
+                <Text style={styles.nextStepsHeadline}>Deine nächsten Schritte</Text>
+                <Text style={styles.nextStepsSub}>Alle Fortschritte anzeigen</Text>
+            </View>
+            <View style={styles.confirmationCardScrollView}>
+                <FlatList
+                    data={Data}
+                    renderItem={({item}) => <ConfirmationCard subject={item.subject} name={item.name} headline={item.headline} body={item.body} linkText={item.linkText}/>}
+                    keyExtractor={item => item.id}
+                    horizontal
+                    showsHorizontalScrollIndicator={false}
+                    ItemSeparatorComponent={() => <View style={{width: 16}}/>}
+                />
+            </View>
+            <View style={{height: 20}}/>
+            <View style={styles.nextAppointment}>
+                <Text style={styles.nextAppointmentText}>Nächster Termin</Text>
+            </View>
+            <AppointmentCard/>
```

Comment:
There isn't any space between the bottom of the screen the the ``AppointmentCard``. Please add it. To ensure space for all elements of the page you can simply give the pages CSS a padding, best would be ``padding: 20`` since this is what I already used for the ``files`` page.

Reply:
added padding of 20 to the bottom of the page